services/mac_manager/src/main/java/com/futurewei/alcor/macmanager/pool/impl/RandomMacPoolImpl.java

```diff
@@ -45,16 +45,16 @@
     private static final float MIDDLE_LOAD_FACTOR = 0.5f;
     private static final float HIGH_LOAD_FACTOR = 0.75f;
 
-    @Value("${low-request-numbers: 10}")
+    @Value("${macmanager.pool.low-request-numbers: 10}")
```

Comment:
I think we should add those parameters to application.properties and yaml file.

Reply:
ok